### PR TITLE
Fix volume_glance_metadata

### PR DESCRIPTION
**Pull request by aguzikova**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.storage.cinder_database
cloudferrylib/os/storage/cinder_database.py:229: [W0125(using-constant-test), CinderStorage._filter_quotas_list] Using a conditional statement with a constant value

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.storage.cinder_database
cloudferrylib/os/storage/cinder_database.py:229: [W0125(using-constant-test), CinderStorage._filter_quotas_list] Using a conditional statement with a constant value

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by ntymtsiv** [comment]
+1, tested on devlab, works good, volume_image_metadata migrated successfully (from icehouse to juno). Found that volumes metadata is not migrated. Created separate bug.

File: cloudferrylib/os/actions/cinder_database_manipulation.py [hunk 1]
```diff
@@ -173,6 +174,75 @@ def __init__(self, *args, **kwargs):
...
+                utils.IDENTITY_RESOURCE)
+            imgs[pos] = self.read_images_info(pos)
+
+        for img in imgs[SRC]:
+            for dst_img in imgs[DST]:
+                skip_fields = ['id', 'resource', 'owner']
+                for f in img['image'].keys():
+                    if f not in skip_fields and \
+                            img['image'][f] != dst_img['image'][f]:
+                        print img['image'][f], ' VS ', dst_img['image'][f]
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/cinder_database_manipulation.py]: Use logs instead

**Comment by roman-verchikov** [comment]
Please provide description and modify pull request title to include corresponding ticket, we follow pattern `SAIC-<issue number> <Short description>` and then long description below

**Comment by roman-verchikov** [comment]
It's quite difficult to grasp the purpose of the patch, most confusing part for me is volume metadata in glance image (???), please clarify the purpose of the patch in description.

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by roman-verchikov** [comment]
continued in https://github.com/MirantisWorkloadMobility/CloudFerry/pull/647

**Pull request closed without merging**